util/util.go

```diff
@@ -261,6 +261,22 @@ func TimestampWithinLimit(latest time.Time, ts string, limit time.Duration) bool
 	return deadline.After(latest)
 }
 
+// TimestampAfterTimestamp returns true if timestamp1 is after timestamp2. It returns false otherwise or if either
+// timestamp cannot be parsed.
```

Comment:
Sorry for the late comment, but this bothers me.  If you ask the question with one or two invalid timestamps, you get an answer as if they were valid.  If you ask the question with the (invalid) timestamps reversed, you get an apparently valid answer with the opposite conclusion, which might cause you to take a whole different path in the code.  Neither is right, maybe, but having such a "random" outcome seems like a good source of unexpected behavior.  Perhaps this should return (bool, error)?

Reply:
My soft preference is for the way I had it (mostly because it is convenient and mirrors the existing `TimestampAfterTimeout` and `TimestampWithinLimit` functions, but I can respect the complaint.

Changed the function to return an error and updated its caller and unit tests.